### ISSUE TITLE
Issue: Plugin "Specflow Rider" is incompatible with this installation

### ISSUE THREAD

**Issue by igladun** [kanekotic/Specflow.Rider#3]
Hello,
Could you please help me to build and install this plugin? After building and trying to install, I get error "Plugin "Specflow Rider" is incompatible with this installation"
Rider version is 2017.1.2
Plugin's source is the most recent (20/10/2017)

**Comment by alvarolorentedev** [comment]
yes i am still working on it :) . So i need to extend the current endpoint for rider so the connection to the environment is still not correct. I want to work on this this week, so if you can wait till next week i will try to have a first binary that you can install released.

**Comment by Xazen** [comment]
That's awesome! I am also looking forward to give it a try.
